src/mono/wasm/runtime/driver.c

```diff
@@ -502,6 +512,8 @@ mono_wasm_assembly_get_entry_point (MonoAssembly *assembly)
 	uint32_t entry = mono_image_get_entry_point (image);
 	if (!entry)
 		return NULL;
+	
+	mono_domain_ensure_entry_assembly (root_domain, assembly);
```

Comment:
This looks good other than it still won't get called on the blazor side.  Their entry point starts at: https://github.com/dotnet/aspnetcore/blob/master/src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts#L54
I'm not sure what the best solution there is.

Reply:
For now it shouldn't block this.